pinpoint-hbase/build/scripts/check-table.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+readarray -t queries < ${BASE_DIR}/hbase-create.hbase
+table_names=("AgentInfo" "AgentStatV2" "ApplicationStatAggre" "ApplicationIndex" "AgentLifeCycle" "AgentEvent" "StringMetaData" "ApiMetaData" "SqlMetaData_Ver2" "TraceV2" "ApplicationTraceIndex" "ApplicationMapStatisticsCaller_Ver2" "ApplicationMapStatisticsCallee_Ver2" "ApplicationMapStatisticsSelf_Ver2" "HostApplicationMap_Ver2")
+
+idx=0
+
+for query in "${queries[@]}"; do
+    if [ "$query" == "" -o "$query" == "list" -o "$query" == "exit" ];then
+        continue
+    fi
+
+    if echo -e "exists '${table_names[idx]}'" | ${HBASE_HOME}/bin/hbase shell 2>&1 | grep -q "does exist" 2>/dev/null
+    then 
+        echo "'${table_names[idx]}' table already exists"
+        idx=$idx+1
+        continue
+    else
+        echo "create '${table_names[idx]}' table"
+        echo "$query" | ${HBASE_HOME}/bin/hbase shell 2>&1
+        idx=$idx+1
+    fi
+done
```

Comment:
I don't this it's necessary to check the table one by one. since it's going to be brought up as a container, only checking one table (etc. AgentInfo) would be enough.

Also running `hbase-create.hbase` when there isn't that `one table` would be better, since this way we don't have to care about changed or added tables.